src/lib/journey/callbacks/terms-and-conditions/terms-conditions.svelte

```diff
@@ -21,7 +25,9 @@
    ************************************************************************* */
   const Checkbox = checkAndRadioType === 'standard' ? Standard : Animated;
   const inputName = callback?.payload?.input?.[0].name || `terms-${idx}`;
-  const terms = callback.getTerms();
+  const termsLinkHtml = `<a href=${$links?.termsAndConditions} target="_blank">${interpolate(
```

Comment:
What should we do in cases where the `links.termsAndConditions` is not provided? As it is now, I feel like this could lead to hidden bugs since we are not conditionally rendering the link on the presence of the URL. The result would be a visually correct, but non-functional link.

Just "thinking out loud" here, but should we render an error if the URL is not provided? That way when the developer is testing, it's obvious the URL is missing. Or, should we just not rendering the message at all? Hmm ...

Reply:
Good point. I think we should render an error if no URL is provided, rather than silently displaying nothing. It feels like there are risks involved if we decide to display nothing and this slips through to a customer's deployed app, since users will not have a chance to read what they are accepting, potentially leading to legal issues. Let's discuss further on our call today as you say.